src/main/ml-modules/root/com.marklogic.smart-mastering/merging.xqy

```diff
@@ -89,7 +89,26 @@ declare function merging:rollback-merge(
   $retain-rollback-info as xs:boolean
 ) as empty-sequence()
 {
-  impl:rollback-merge($merged-doc-uri, $retain-rollback-info)
+  impl:rollback-merge($merged-doc-uri, $retain-rollback-info, fn:true())
```

Comment:
Added boolean for blocking future merges when rolling back. defaults to true and blocks future merges.